ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/analytics/model/TutorialAnalyticsScreenEvent.kt

```diff
@@ -4,7 +4,11 @@ import org.cru.godtools.analytics.model.AnalyticsScreenEvent
 import org.cru.godtools.tutorial.PageSet
 import java.util.Locale
 
-class TutorialAnalyticsScreenEvent(tutorial: PageSet, page: Int) :
+class TutorialAnalyticsScreenEvent(val tutorial: PageSet, page: Int) :
     AnalyticsScreenEvent("${tutorial.analyticsBaseScreenName}-${page + 1}", Locale.getDefault()) {
-    override val adobeSiteSection get() = ADOBE_SITE_SECTION_TUTORIAL
+    override val adobeSiteSection
+        get() = when (tutorial.analyticsBaseScreenName) {
+            BASE_SCREEN_NAME_ON_BOARDING -> ADOBE_SITE_SECTION_ON_BOARDING
+            else -> ADOBE_SITE_SECTION_TUTORIAL
+        }
```

Comment:
you know that `tutorial` is an enum, there isn't a need to extract string constants to switch between the different `PageSets`

Reply:
@frett I am not sure what you mean by this comment.

Reply:
```kotlin
get() = when (tutorial) {
    ONBOARDING -> ADOBE_SITE_SECTION_ON_BOARDING
    TUTORIAL -> ADOBE_SITE_SECTION_TUTORIAL
    else -> null
}
```

Reply:
AH I see

Reply:
I don't think you understood what I was saying here at all

Reply:
this suggested change was the reason that the `BASE_SCREEN_NAME_*` constants weren't needed